src/components/Form/PhotoUpload.vue

```diff
@@ -15,16 +16,16 @@
       <template v-slot:header="scope">
         <q-card flat>
           <q-spinner v-if="scope.isUploading" class="q-uploader__spinner" />
-          <div class="text-subtitle2 absolute-bottom text-center">
-            <div class="q-uploader__title">{{$t('Drop a Photo')}}</div>
+          <div class="text-subtitle2">
+            <div class="q-uploader__title">{{$t(defaultText)}}</div>
             <div class="q-uploader__subtitle">
               {{ scope.uploadSizeLabel }} /
               {{ scope.uploadProgressLabel }}
             </div>
           </div>
           <q-btn v-if="scope.canAddFiles" type="a" icon="add_box" round dense flat>
             <q-uploader-add-trigger />
-            <q-tooltip>{{$t('Choose Photo')}}</q-tooltip>
+            <q-tooltip>{{$t('Your Logo')}}</q-tooltip>
```

Comment:
The ToolTip is shown on hover at the "+" Icon. 

By clicking the "+" Icon, the File-Selector Dialog of the Browser opens. So I would say, the text should contain "Choose" or "select"